studio/.env.docker

```
@@ -12,4 +12,7 @@ NEXT_PUBLIC_SENTRY_DSN=APP_NEXT_PUBLIC_SENTRY_DSN
 NEXT_PUBLIC_SENTRY_CLIENT_SAMPLE_RATE=APP_NEXT_PUBLIC_SENTRY_CLIENT_SAMPLE_RATE
 NEXT_PUBLIC_SENTRY_CLIENT_TRACES_SAMPLE_RATE=APP_NEXT_PUBLIC_SENTRY_CLIENT_TRACES_SAMPLE_RATE
 NEXT_PUBLIC_SENTRY_CLIENT_REPLAYS_ON_ERROR_SAMPLE_RATE=APP_NEXT_PUBLIC_SENTRY_CLIENT_REPLAYS_ON_ERROR_SAMPLE_RATE
-NEXT_PUBLIC_SENTRY_CLIENT_REPLAYS_SESSION_SAMPLE_RATE=APP_NEXT_PUBLIC_SENTRY_CLIENT_REPLAYS_SESSION_SAMPLE_RATE
\ No newline at end of file
+NEXT_PUBLIC_SENTRY_CLIENT_REPLAYS_SESSION_SAMPLE_RATE=APP_NEXT_PUBLIC_SENTRY_CLIENT_REPLAYS_SESSION_SAMPLE_RATE
+
+# tracking cookies
+NEXT_PUBLIC_TRACKING_COOKIE_DOMAIN_NAME=APP_NEXT_PUBLIC_TRACKING_COOKIE_DOMAIN_NAME
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)